build.gradle

```diff
@@ -20,6 +20,11 @@ task wrapper(type: Wrapper) {
     gradleVersion = '4.6'
 }
 
+dependencies {
+    // https://mvnrepository.com/artifact/org.bouncycastle/bcprov-jdk15on
+    implementation group: 'org.bouncycastle', name: 'bcprov-jdk15on', version: '1.60'
```

Comment:
Would it also be possible to use very much smaller `bcpkix-jdk15on` artifact?

Reply:
I would also prefer to create a shadow jar including BC using https://imperceptiblethoughts.com/shadow/